langchain/schema/messages.py

```diff
@@ -77,6 +77,13 @@ def lc_serializable(self) -> bool:
         """Whether this class is LangChain serializable."""
         return True
 
+    # Cannot type this since it returns something we can't import
```

Comment:
could maybe gate on TYPE_CHECKING but maybe not

Reply:
yeah good call